source/release-notes/release_3_7_0.rst

```diff
@@ -0,0 +1,40 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _release_3_7_0:
+
+3.7.0 Release Notes
+===================
+
+
+This section shows the most relevant improvements and fixes in version 3.2.3. More details about these changes are provided in each component changelog.
+
+- `wazuh/wazuh <https://github.com/wazuh/wazuh/blob/v3.7.0/CHANGELOG.md>`_
+- `wazuh/wazuh-api <https://github.com/wazuh/wazuh-api/blob/v3.7.0/CHANGELOG.md>`_
+- `wazuh/wazuh-ruleset <https://github.com/wazuh/wazuh-ruleset/blob/v3.7.0/CHANGELOG.md>`_
+- `wazuh/wazuh-kibana-app <https://github.com/wazuh/wazuh-kibana-app/blob/v3.7.0-6.4.1/CHANGELOG.md>`_
+- `wazuh/wazuh-kibana-splunk <https://github.com/wazuh/wazuh-splunk/blob/v3.7.0-7.1.3/CHANGELOG.md>`_
```

Comment:
Please change the Wazuh apps (KIbana and Splunk) tags to the following:
- Kibana: `v3.7.0-6.4.2`
- Splunk: `v3.7.0-7.2.0`